src/Text/Atom/Feed/Import.hs

```diff
@@ -253,7 +253,11 @@ pContent e =
   case pAttr "type" e of
     Nothing -> return (TextContent (elementTexts e))
     Just "text" -> return (TextContent (elementTexts e))
-    Just "html" -> return (HTMLContent (elementTexts e))
+    Just "html" -> 
+      case children e of
+        [] -> return (TextContent "")
+        [c] -> return (HTMLContent c)
+        _ -> Nothing
```

Comment:
Is it possible that there are more than one element? Would it make sense to match on `c :: _` above instead?

Reply:
Hmm definitely seems possible, but I'm not sure what the "correct" behavior would be in that case for this update then. I'm essentially mimicking the XHTML behavior right below this in the code. I've also only seen a limited sample size of one for where I needed this behavior so far. 

The other option would be to make `HTMLContent` wrap a `[Element]` and then just store the children there? Is there an RFC spec that this should be conforming to that you know of that we could look up the "right" answer here?

Reply:
@bergmark Gentle ping on this whenever you have time!

Reply:
I don't know how to figure it out...

We could merge this as is and change it later if anything pops up, what do you think?

Reply:
Sounds like a plan. I can also run it on my own for a bit and see if anything comes up that goes against this assumption.